### PR TITLE
Fix unit tests that fail due to jenkins CSS changes

### PR DESCRIPTION
**Pull request by mikecirioli**
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

HTML element attribute id names have changed in newer versions of Jenkins (2.33x.x), causing a few unit tests to break.  This PR updates and fixes them.

### CODE REVIEW

**Review by raul-arabaolaza — APPROVED** [review]

**Review by car-roll — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -67,7 +67,12 @@
                 <groupId>org.jenkins-ci</groupId>
                 <artifactId>symbol-annotation</artifactId>
                 <version>1.23</version>
-            </dependency>  
+            </dependency>
+            <dependency>
+                <groupId>com.google.errorprone</groupId>
+                <artifactId>error_prone_annotations</artifactId>
+                <version>2.10.0</version>
+            </dependency>
```
Comment (jtnord) [inline pom.xml:75]: was excluded for `script-security` do we need to version manage this or should it be excluded _if_ coming from core via guava?

File: src/test/java/org/jenkinsci/plugins/configfiles/ConfigFilesSEC1253Test.java [hunk 2]
```diff
@@ -55,7 +55,7 @@ public void regularCaseStillWorking() throws Exception {
         assertThat(store.getConfigs(), hasSize(1));
 
         HtmlPage configFiles = wc.goTo("configfiles");
-        HtmlAnchor removeAnchor = configFiles.getDocumentElement().getFirstByXPath("//a[contains(@onclick, 'removeConfig?id=" + CONFIG_ID + "')]");
+        HtmlAnchor removeAnchor = configFiles.getDocumentElement().getFirstByXPath("//a[contains(@data-url, 'removeConfig?id=" + CONFIG_ID + "')]");
```
Comment (jtnord) [inline src/test/java/org/jenkinsci/plugins/configfiles/ConfigFilesSEC1253Test.java:58]:
```suggestion
        HtmlAnchor removeAnchor = configFiles.getDocumentElement().getFirstByXPath("//a[contains(@data-url, 'removeConfig?id=" + CONFIG_ID + "')|contains(@onclick, 'removeConfig?id=" + CONFIG_ID + "')]");
```

should (if I got my XPath correct) have allowed to fix a PCT issue without bumping the core to a non LTS making upgrades for LTS users harder.

**Review by jtnord — APPROVED** [review]

File: src/test/java/org/jenkinsci/plugins/configfiles/ConfigFilesSEC1253Test.java (hunk 2 above)
Comment (mikecirioli) [reply src/test/java/org/jenkinsci/plugins/configfiles/ConfigFilesSEC1253Test.java:58]: basically, your saying to make this test backwards compatible with older jenkins?

Comment (jtnord) [reply src/test/java/org/jenkinsci/plugins/configfiles/ConfigFilesSEC1253Test.java:58]: I am saying you could have done that.  as it is a test only issue and the plugin will work after an upgrade of Jenkins, the current way is not making it any harder for users (there would be no broken plugins post upgrade) so it is fine as is.  (hence my approval)

**Comment by ampuscas** [comment]
@mikecirioli starting from your PR, I have created the PR #199 where I merged master and updated some dependencies because the build was failing due to Maven enforcer rule, so this PR becomes obsolete.

**Comment by basil** [comment]
Suggest closing in favor of #200.

**Pull request merged**